e2e/libs/workload/workload.py

```diff
@@ -130,11 +130,15 @@ def check_pod_data_checksum(expected_checksum, pod_name, file_name, data_directo
         command=cmd_get_file_checksum, stderr=True, stdin=False, stdout=True,
         tty=False)
 
+    logging(f"Checked {pod_name} file {file_name} checksum: \
+        Got {file_path} checksum = {actual_checksum} Expected checksum = {expected_checksum}")
+
     if actual_checksum != expected_checksum:
-        message = f"Got {file_path} checksum = {actual_checksum} \
-            Expected checksum = {expected_checksum}"
+        message = f"Checked {pod_name} file {file_name} checksum failed. \
+        Got {file_path} checksum = {actual_checksum} Expected checksum = {expected_checksum}"
```

Comment:
nit: Checksum verification failed for file {file_name} in pod {pod_name}. Expected checksum = {expected_checksum}, but got {actual_checksum} for file at {file_path}.

Reply:
I think we have a coding convention logging messages should start with `V-ing` or `V-ed`?

https://github.com/longhorn/longhorn/wiki/coding-convention#logging
https://github.com/longhorn/longhorn-tests/pull/1520#discussion_r1322681680